### PR TITLE
Add similar courses in course page

### PR DESCRIPTION
**Pull request by JortvD**
Created to solve #1726

### CODE REVIEW

**Comment by Koen1999** [comment]
Is there a way to populate `$similarCoursesFrom` in the model?

**Comment by JortvD** [comment]
> Is there a way to populate `$similarCoursesFrom` in the model?

I have to preface that I'm a bit new to Doctrine ORM, but isn't it the case that `$similarCoursesFrom` is populated due to the fact it is the inverse of `$similarCoursesTo`? So if entries are added to the SimilarCourse table it will populate the related collections for those courses.

File: module/Education/src/Service/Course.php [hunk 1]
```diff
@@ -485,13 +485,60 @@ public function getCourseForm(?CourseModel $course = null): CourseForm
...
+     * } $data
      */
-    public function saveCourse(CourseModel $course): bool
+    public function saveCourse(array $data): CourseModel
     {
+        $course = new CourseModel();
+        $course->setCode($data['code']);
+        $course->setName($data['name']);
+
+        $similarCoursesCodes = explode(',', $data['similar']);
```
Comment (Koen1999) [inline module/Education/src/Service/Course.php:502]: I see you added it to the form, good! Will be quite some manual work though.

Comment (JortvD) [reply module/Education/src/Service/Course.php:502]: Yep, but that's the job :)

**Review by tomudding — APPROVED** [review]
Looks good @JortvD! Because you will be using this interface it is fine for now, however, I do think we should make the selection process of courses easier in the future. For example, with an auto-complete search function.

**Pull request merged**